File: app/src/main/java/com/seafile/seadroid2/framework/model/ResultModel.java
package com.seafile.seadroid2.framework.model;

public class ResultModel {
    public boolean success;
    public String error_msg;

    @Override
    public String toString() {
        return "ResultModel{" +
                "success=" + success +
                ", error_msg='" + error_msg + '\'' +
                '}';
    }
}

File: app/src/main/java/com/seafile/seadroid2/framework/model/ResultModel.kt
package com.seafile.seadroid2.framework.model

open class ResultModel {
    @JvmField
    var success: Boolean = false

    @JvmField
    var error_msg: String? = null

    override fun toString(): String {
        return "ResultModel(success=$success, error_msg=$error_msg)"
    }
}
